cmd/kubelet/app/server.go

```diff
@@ -581,6 +584,10 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, stopCh <-chan
 			clientCertificateManager.SetCertificateSigningRequestClient(kubeClient.CertificatesV1beta1().CertificateSigningRequests())
 			clientCertificateManager.Start()
 		}
+		dynamicKubeClient, err = dynamic.NewForConfig(clientConfig)
+		if err != nil {
+			glog.Errorf("Failed to initialize dynamic KubeClient: %v", err)
```

Comment:
It seems that we use `Warningf` for the other clients.

Reply:
Done.